packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -344,9 +380,11 @@ class ReactSixteenAdapter extends EnzymeAdapter {
     };
   }
 
-  createShallowRenderer(/* options */) {
+  createShallowRenderer(options) {
```

Comment:
```suggestion
  createShallowRenderer(options = {}) {
```
this keeps the `.length` the same.